README.md

```diff
@@ -171,6 +172,11 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
 
 **[⬆️ Back to Top](#table-of-contents)**
 
+## Cloud Platforms
+  * [Warren Cloud Platform](https://warren.io/) - Managed cloud platform that delivers CI upgrades with no downtime ~twice a week. They offer Free Tier for managed private cloud deployments smaller than 256 GB RAM.
```

Comment:
It seems like this would go well with some of the existing categories we do not need a new one

Reply:
But also I don't think this is a SaaS.